modules/Microsoft.DigitalTwins/digitalTwinsInstances/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,166 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Event Hub Namespace to create.')
+param eventHubNamespaceName string
+
+@description('Required. The name of the Event Hub to create.')
+param eventHubName string
+
+@description('Required. Service Bus name')
+param serviceBusName string
+
+@description('Required. Service Bus topic name.')
+param serviceBusTopicName string
+
+@description('Required. Event Grid Domain name.')
+param eventGridDomainName string
+
+@description('Required. Event Grid Topic name.')
+param eventGridTopicName string
+
+var addressPrefix = '10.0.0.0/16'
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+    name: virtualNetworkName
+    location: location
+    properties: {
+        addressSpace: {
+            addressPrefixes: [
+                addressPrefix
+            ]
+        }
+        subnets: [
+            {
+                name: 'defaultSubnet'
+                properties: {
+                    addressPrefix: addressPrefix
+                    serviceEndpoints: [
+                        {
+                            service: 'Microsoft.KeyVault'
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+}
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+    name: managedIdentityName
+    location: location
+}
+
+resource privateDNSZone 'Microsoft.Network/privateDnsZones@2020-06-01' = {
+    name: 'privatelink.vaultcore.azure.net'
```

Comment:
Is this really `vaultcore`? It's an honest question, but I'd have assumed it's something special for Digital Twins.

Reply:
like [`privatelink.digitaltwins.azure.net`](https://learn.microsoft.com/en-us/azure/digital-twins/how-to-enable-private-link?tabs=portal%2Cportal-2#add-a-private-endpoint-to-an-existing-instance)

Reply:
Thanks just verified that its [privatelink.digitaltwins.azure.net](privatelink.digitaltwins.azure.net). The target sub resource is "API". I will update this.